sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/DeltaKey.java

```diff
@@ -0,0 +1,49 @@
+package com.bazaarvoice.emodb.sor.db.astyanax;
+
+import com.netflix.astyanax.annotations.Component;
+
+import java.util.UUID;
+
+public class DeltaKey {
+
+    private @Component(ordinal=0) UUID _changeId;
+    private @Component(ordinal=1) Integer _block;
```

Comment:
I'm not sure actually. I made it Integer because that is the way it was done in the Astyanax docs. Definitely something i'll look into.